File: Rasha_Toine/Rasha/QLForm/src/qls/ast/style/FontSize.java
package qls.ast.style;

import ql.ast.literal.IntegerLiteral;
import qls.visiting.StyleVisitor;


public class FontSize extends StyleProperty {
	private IntegerLiteral size;

	//constructor
	public FontSize(IntegerLiteral size) {
		this.size = size;
	}

	public int getSize() {
		return size.getValue().getValue();
	}

	@Override
	public <T, U> T accept(StyleVisitor<T, U> visitor, U ctx) {
		return visitor.visit(this, ctx);
	}
}

File: Erik_Vincent/Assignment1/Assignment1/Model/QLS/AST/Style/FontSize.cs
using Assignment1.Model.QL.AST;

namespace Assignment1.Model.QLS.AST.Style
{
    public class FontSize : ASTNode, IStyle
    {
        public int Value { get; }

        private int _value;

        public FontSize(int lineNumber, int value)
        {
            _lineNumber = lineNumber;
            _value = value;
        }

        public void Accept(IStyleVisitor visitor) => visitor.Visit(this);
    }
}
